CommunityCall/README.md

```diff
@@ -1,6 +1,9 @@
-# PowerShell Community Call
+# PowerShell Core Community Call
+
+The PowerShell Core Community Call is held on the 3rd Thursday of every month.
+Use [this ICS file](RecurringCommunityCall.ics) (right-click Raw -> Save Target/Link As in order to download the file) to put this recurring event on your calendar. 
```

Comment:
Can you make the menu options more clear using backticks?

`Raw` -> `Save Target/Link As`